src/rev_ai/models/error.py

```diff
@@ -0,0 +1,112 @@
+class ProblemDetail:
+	def __init__(
```

Comment:
Should we use tabs or spaces? Ask @JDongian.

Reply:
spaces.
The same arguments apply for any language (except Make, where it is not an option)
* pastability
* everyone does it
* easier to align funky function arguments

Reply:
Everything should be spaces now. :+1: